bigquery/bigquery.go

```diff
@@ -234,9 +234,9 @@ func runWithRetry(ctx context.Context, call func() error) error {
 func runWithRetryExplicit(ctx context.Context, call func() error, allowedReasons []string) error {
 	// These parameters match the suggestions in https://cloud.google.com/bigquery/sla.
```

Comment:
Sounds like with those changes, it is not going to match the suggestions found in https://cloud.google.com/bigquery/sla

```
"Back-off Requirements" means, when an error occurs, the Customer Application is responsible for waiting for a period of time before issuing another request. This means that after the first error, there is a minimum back-off interval of 1 second and for each consecutive error, the back-off interval increases exponentially up to 32 seconds.
```

...**a minimum back-off interval of 1 second**...

Reply:
Ahh yes.  I'll back out the retry changes.